YukkiMusic/plugins/tools/reload.py

```diff
@@ -21,78 +21,61 @@
                                          language)
 from YukkiMusic.utils.formatters import alpha_to_int
 
-### Multi-Lang Commands
+# Multi-Lang Commands
 RELOAD_COMMAND = get_command("RELOAD_COMMAND")
 RESTART_COMMAND = get_command("RESTART_COMMAND")
 
 
-@app.on_message(
-    filters.command(RELOAD_COMMAND)
-    & filters.group
-    & ~filters.edited
-    & ~BANNED_USERS
-)
+@app.on_message((((filters.command(RELOAD_COMMAND) & filters.group)) & ~BANNED_USERS))
 @language
 async def reload_admin_cache(client, message: Message, _):
     try:
         chat_id = message.chat.id
-        admins = await app.get_chat_members(
-            chat_id, filter="administrators"
-        )
+        admins = await app.get_chat_members(chat_id, filter="administrators")
```

Comment:
Did you read documentations? refer to [1](https://docs.pyrogram.org/api/methods/get_chat_members) and [2](https://docs.pyrogram.org/api/enums/ChatMembersFilter#pyrogram.enums.ChatMembersFilter)